web/beacon-app/package.json

```diff
@@ -55,7 +55,7 @@
     "@radix-ui/react-icons": "^1.3.0",
     "@radix-ui/react-tooltip": "^1.0.5",
     "@reduxjs/toolkit": "^1.9.2",
-    "@rotational/beacon-core": "^2.3.8",
+    "@rotational/beacon-core": "^2.3.10",
```

Comment:
This version added the disabled HTML attribute to the Textfield component and also applied style when disabled or isDisabled props is added